templates/3_Pixie_Cluster_Cells.ipynb

```diff
@@ -27,6 +27,7 @@
    "outputs": [],
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.        fc.create_file_chooser(base_dir, dir_to_assign="cell_clustering_params_dir", file_to_assign="cell_clustering_params_path")
</pre><p>Is there any reason the user would have a different file path than the default <code>/base_dir/pixie/example_pixel_output_dir/cell_clustering_params.json</code>? I feel like this could be an automatic assignment rather than having the user find and select a nested file like this.  Maybe selecting the pixel_output_dir would make more sense, e.g. if the user had multiple pixel clustering runs.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/1063/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
Don't think so, good idea to just leave it as an automatic assignment.

Reply:
So to load the cell_clustering_params.json file the user will need to specify the path to the pixel_output_dir created in the previous notebook. Still agreed we don't need them to explicitly specify the cell_clustering_params.json file though.